centos_ci.sh

```diff
@@ -201,12 +199,13 @@ function docs_tar_upload() {
   set +x
 
   version=$(cat docs/source/variables.adoc | cut -d' ' -f2 | head -n1)
-  mkdir -p minishift/docs/$version
+  LATEST=latest_test
+  mkdir -p minishift/docs/$version minishift/docs/$LATEST
   cp docs/build/minishift-adoc.tar minishift/docs/$version/
-  ln -s $(pwd)/minishift/docs/$version minishift/docs/latest
+  cp docs/build/minishift-adoc.tar minishift/docs/$LATEST/
```

Comment:
So you basically create both directories (latest and version) and copy them. No symbolic link. Not quite the same, but it will do